src/allencell_ml_segmenter/core/file_input_widget.py

```diff
@@ -214,14 +214,18 @@ def _from_directory_slot(self) -> None:
         self._model.set_input_mode(InputMode.FROM_PATH)
 
     def _update_layer_list(self, event: Optional[NapariEvent] = None) -> None:
```

Comment:
Im confused about this being a method of FileInputWidget.  I know there are reasons and that it would be hard to change now.

Reply:
please link reactor ticket